src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -294,6 +299,7 @@ class BasicSettings extends React.Component {
       : [ { id: '_', value: `-- ${msg.createVmWizardSelectClusterBeforeISO()} --` } ]
     if (enableIsoSelect && !isValidUid(data.isoImage)) {
       isoList.unshift({ id: '_', value: `-- ${msg.createVmWizardSelectISO()} --` })
+      indicators.isoList = !indicators.provisionSource && !indicators.name && 'error'
     }
```

Comment:
Perhaps we should reset the indicator in the `else` block?
Just to be on the safe side - the code seems to work without that.